OpenIPSL/Electrical/Buses/BusExt.mo

```diff
@@ -26,19 +26,20 @@ model BusExt
         origin={0,0},
         extent={{-4,-60},{4,60}},
         rotation=0)));
-  Real V(start=V_0) "Bus voltage magnitude (pu)";
-  Real angle(start=angle_0) "Bus voltage angle (deg)";
-  parameter Real V_0=1 "Voltage magnitude (pu)"
+  SI.PerUnit v(start=v_0) "Bus voltage magnitude (pu)";
+  SI.Conversions.NonSIunits.Angle_deg angle(start=angle_0) "Bus voltage angle";
+  parameter SI.PerUnit v_0=1 "Voltage magnitude (pu)"
     annotation (Dialog(group="Power flow data"));
-  parameter Real angle_0=0 "Voltage angle (deg)"
+  parameter SI.Conversions.NonSIunits.Angle_deg angle_0=0 "Voltage angle"
     annotation (Dialog(group="Power flow data"));
-  parameter Real V_b=130 "Base voltage (kV)"
+  parameter SI.Voltage V_b=130e3 "Base voltage"
```

Comment:
@dietmarw here you did not used the command to display the unit in kV. Was it intentional?

Reply:
Good catch. This was an oversight. Will fix it right away.

Reply:
Anything else of can this be merged?